spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:example) do
+    @user = User.create(name: 'Dev', photo: 'myPhoto', bio: 'myBio', post_counter: 0)
+    @post = Post.create(title: 'myTitle', text: 'Hello', author: @user, comments_counter: 0, likes_counter: 0)
+    @comment = Comment.create(text: 'comment', post: @post, author: @user)
+  end
+
+  it 'have correct user' do
+    expect(@comment.author_id).to eq(@user.id)
+  end
+
+  it 'have correct post' do
+    expect(@comment.post_id).to eq(@post.id)
+  end
+
+  it 'test if comment counter is updated' do
+    Comment.create(text: 'comment_1', post: @post, author: @user)
+    expect(@post.comments_counter).to eq 2
+  end
+end
```

Comment:
- [ ] The entire `Comment`  model tests are failing as you can see in the attached image below. Please make the tests green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/229285205-5dc99c95-37a6-47f2-85a6-82b84432c6ff.png)
**HINT:** I believe this has something to do with your having `posts_id` instead of `post_id` column in `comments` database table